src/utilities/getIndicatorTableData.js

```diff
@@ -80,6 +80,16 @@ const getIndicatorRowColumnValue = (row, column, period) => {
   return rowColumnValue;
 };
 
+// eslint-disable-next-line no-unused-vars
```

Comment:
you like unused vars a lot !

Reply:
Yeah they're just placeholders waiting desktop fixes. Theyll get removed soon.